### PR TITLE
manifest: update fw-nrfconnect-mcuboot

### PR DESCRIPTION
**Pull request by sigvartmh**
Update fw-nrfconnect-mcuboot revision to refrence
https://github.com/NordicPlayground/fw-nrfconnect-mcuboot/pull/18

Signed-off-by: Sigvart Hovland <sigvart.hovland@nordicsemi.no>

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ manifest:
       revision: 3757f8cfe6ffe2c2b537815bb9ca1ace85882df8
     - name: fw-nrfconnect-mcuboot
       path: mcuboot
-      revision: 40ab097e616bcd393fc7afccce7704c76f5c6c4e
+      revision: pull/18/head
```
Comment (nvlsianpu) [inline west.yml]: 0341ae4f75e22e18aae9dfa42dd104e3352cf2b4

**Comment by nvlsianpu** [comment]
CI passed -counterpart is merged already.

**Comment by sigvartmh** [comment]
Done

File: west.yml (hunk 1 above)
Comment (sigvartmh) [reply west.yml]: Fixed @nvlsianpu

**Pull request merged**